doc/Learn/Tutorials/DependencyInjection/HowTo-DependencyInjectionSetup.md

```diff
@@ -3,10 +3,7 @@ uid: Learn.Tutorials.DependencyInjection.HowToDependencyInjection
 ---
 # How-To: Use Services with Dependency Injection
 
-Dependency Injection (DI) is an important design pattern for building loosely-coupled software that allows for maintainability and testing. This tutorial will walk you through how to register services so that they can be consumed throughout your application.
-
-> [!TIP]
-> This guide assumes you used the Uno.Extensions `dotnet new unoapp-extensions` template to create the solution. Instructions for creating an application from the template can be found [here](xref:Overview.Extensions)
+Dependency Injection (DI) is an important design pattern when building loosely-coupled software that allows for maintainability and testing. This tutorial will walk you through how to register services so that they can be consumed throughout your application.
```

Comment:
Do we not need the !TIP any more?

Reply:
@nickrandolph I observed the tips weren't applied consistently to the docs which require them. Is it really necessary to keep these tips when its already implied that uno extensions is needed?

Reply:
Yeh, we do - people will land on this page directly, so we need to give them all steps necessary to be successful